test/post.ts

```diff
@@ -169,7 +169,7 @@ test('throws when form body is not a plain object or array', async t => {
 });
 
 // See https://github.com/sindresorhus/got/issues/897
-test.failing('the `json` payload is not touched', withServer, async (t, server, got) => {
```

Comment:
Can you update the test to also check context, form, body, and hooks? Just to ensure we never regress the other properties either.

Reply:
Done. The tests for `context` already exist in the `arguments.ts` file.